### ISSUE TITLE
[Typescript] event beforeoperations/afteroperations not assignable on RasterSource

### ISSUE THREAD

**Issue by imagoiq** [openlayers/openlayers#12573]
**Describe the bug**
`beforeoperations` and `afteroperations` events are not assignable on a raster source.

```
Argument of type 'string' is not assignable to parameter of type '("error" | "change" | "propertychange" | "imageloadstart" | "imageloadend" | "imageloaderror")[]'.
```

**To Reproduce**
```
import { Raster as RasterSource } from 'ol/source';
const source = new RasterSource({ // sources, operations });
source.on('beforeoperations', (event: any) => { // Do something });
```

**Expected behavior**
Be allowed to use those events 😀

**Comment by ahocevar** [comment]
Thanks for reporting this, @imagoiq. Will be fixed with #12575.

**Comment by imagoiq** [comment]
Thanks to you!
